modules/core/04-channel/keeper/upgrade.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"reflect"
 
+	"golang.org/x/exp/slices"
```

Comment:
feat branch is still on go1.20 I guess. 

Its probably fine to keep this and fix it in the merge

Reply:
we could just replace the compare with a direct comparison of ConnectionHops[0] since that's all we actually have now.